extensions/positron-python/python_files/posit/positron/positron_ipkernel.py

```diff
@@ -148,10 +152,38 @@ def view(self, line: str) -> None:
                 raise UsageError(f"{e.args[0]}. Did you quote the title?") from e
             raise
 
-        # Find the object.
+        # First try to find the object directly by name
         info = self.shell._ofind(args.object)  # noqa: SLF001
+
         if not info.found:
-            raise UsageError(f"name '{args.object}' is not defined")
+            # Check if the object name is a quoted string and remove quotes if necessary
+            obj_name = args.object
+            if (obj_name.startswith('"') and obj_name.endswith('"')) or (
+                obj_name.startswith("'") and obj_name.endswith("'")
+            ):
+                obj_name = obj_name[1:-1]  # Remove the quotes
+
+            # If not found as a variable, try to evaluate it as an expression
+            try:
+                obj = self.shell.ev(obj_name)
+
+                # Create a similar info object to what _ofind would return
+                class SimpleNamespaceObj:
+                    def __init__(self) -> None:
+                        self.found: bool = False
+                        self.obj: Any = None
+                        self.ismagic: bool = False
+                        self.isalias: bool = False
+                        self.namespace: str = ""
+
+                info = SimpleNamespaceObj()
+                info.found = True
+                info.obj = obj
+                info.ismagic = False
+                info.isalias = False
+                info.namespace = "Interactive"
```

Comment:
We don't need to mock the info object, I think the only thing that's used is `obj`.